test/utils.jl

```diff
@@ -24,10 +24,15 @@ end
 
   xs, y = randn(2,3), rand()
   f34(xs, y) = xs[1] * (sum(xs .^ (1:3)') + y^4)  # non-diagonal Hessian, two arguments
-  dx, dy = diaghessian(f34, xs, y)
-  @test size(dx) == size(xs)
-  @test vec(dx) ≈ diag(hessian(x -> f34(x,y), xs))
-  @test dy ≈ hessian(y -> f34(xs,y), y)
+
+  function broken()
```

Comment:
Cross ref to https://github.com/JuliaDiff/ForwardDiff.jl/pull/538

```suggestion
  # Follow is should work ones we workout what ForwardDiff should do when `Float64` is called on a `Dual`
  # https://github.com/JuliaDiff/ForwardDiff.jl/pull/538
  # else might need a custom overload of `(;;ChainRulesCore.ProjectTo)(::Dual)`
  # When fixed uncomment  the below and delete the broken function
  #==
  dx, dy = diaghessian(f34, xs, y)
  @test size(dx) == size(xs)
  @test vec(dx) ≈ diag(hessian(x -> f34(x,y), xs))
  @test dy ≈ hessian(y -> f34(xs,y), y)
  ==#  
  function broken()
```